app/js/sendDMX.js

```diff
@@ -0,0 +1,102 @@
+const fs = parent.require('fs');
+const DMX = parent.require('dmx');
+const player = parent.require('play-sound')(opts = {});
+const NanoTimer = parent.require('nanotimer');
+const load = parent.require('audio-loader');
+const { Howl, Howler } = parent.require('howler')
+
+let canPlay = false; // bool check for if should go to next sequence
+
+function startCanPlay() {
+    canPlay = true;
+}
+
+function stopCanPlay() {
+    canPlay = false;
+}
+function stopPlaying(showElement) {
+    showElement.getAudio().stop();
+    if (showElement.getUniverse()) {
+        showElement.getUniverse().close();
+    }
+    showElement.getTimer().clearInterval();
+}
+
+function update(showElement) {
+    const index = Math.ceil((new Date() - showElement.getStartTime()) / 50); // HARDCODE
```

Comment:
I'd say deal with this hardcoded value (and the other one csvAddShow) in a later issue.